elements/drag-drop-elements/info.json

```diff
@@ -0,0 +1,14 @@
+{
+    "controller": "pl-drag-drop-element.py",
+    "dependencies": {
+        "nodeModulesScripts": [
+            "jquery-ui-dist/jquery-ui.js"
```

Comment:
Using "jquery-ui.min.js" will be functionally equivalent but slightly more performant.

Reply:
will include this change in next commit